Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -54,17 +54,37 @@ public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_
   }
 
 %{
+    RESULT_BUILDER_TYPES = {
+      'CodeBlock': 'CodeBlockItemList',
+      'MemberDeclBlock': 'MemberDeclList',
+      'CodeBlockItemList': None,
+      'MemberDeclList': None,
+      'PatternBindingList': None,
+    }
```

Comment:
Do you think it makes sense to move this definition to its own file similar to ExpressibleAsConformances.py?